src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -163,6 +171,15 @@ const Conversation: React.FC<ConversationProps> = props => {
 
   const [toastBottom, setToastBottom] = useState(0)
 
+  // Show banner
+  const [showBanner, setShowBanner] = useState(true)
+
+  useEffect(() => {
```

Comment:
at the top of my head, I don't think this component needs to observe any data change to show the Toast (unless the component is never unmounted when we switch between conversations), I would expect something like:

```jsx
useEffect(() => {
  setShowBanner(true) // if we observe conversationID from the deps of useEffect
  const timer = setTimeout(() => setShowBanner(false), 5000)
  return () => clearTimeout(timer)
}, []) // or we use conversationID in the deps that we define at line 131
```
if you observe `conversationID` here you can start the initial state of `showBanner` as false, then once we have `conversationID` changes will be set to true 

another point, can you please check the timer of the banner? I recall talking to Charles from Eigen and he said 3 seconds, not 5

Reply:
The issue is that the `conversationID` and the url changes every time we change a conversation, so we can see that with `selectedConversationID` . I will look into this option.

This task for Force requires 5 seconds, unless some other decision has been made in the meantime?

Reply:
5 seconds 👍 , I guess we just have a different time for Eigen I suppose, maybe because the screen is smaller

regarding updating the component I understand, but still, we don't need this extra parameter since we already have `conversationID` defined in the component 😉

Reply:
I changed it to `conversationID` as suggested.